docs-src/concepts/what-is-cluster-configuration.md

```diff
@@ -0,0 +1,127 @@
+---
+id: what-is-cluster-configuration
+title: What is Cluster configuration?
+sidebar_label: Cluster configuration
+description: Cluster Configuration is the setup and configuration details of your Temporal Cluster, defined using YAML.
+tags:
+  - term
+  - explanation
+---
+
+Cluster configuration is the setup and configuration details of your self-hosted Temporal Cluster, defined using YAML.
```

Comment:
maybe nice to show small example and/or note that the configuration is written in yaml